setup/gcp/installer/forseti_server_installer.py

```diff
@@ -92,6 +92,10 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
                 self.gcp_service_account,
                 self.user_can_grant_roles)
 
+            # Update rule files with default values
+            rule_vals = self.get_rule_default_values()
+            files.update_rule_files(rule_vals, constants.RULES_DIR_PATH)
```

Comment:
This block of code can be self-documenting (drop the comment), and read better as:
```
default_rule_values = def get_default rule_values()
files.update_rule_files(default_rule_values)
```

Reply:
Updated.

Reply:
I don't see the updated change.

Reply:
Just pushed.